scripts/options.js

```diff
@@ -118,6 +117,58 @@ onClickElements.forEach(el => {
 	document.getElementById(el).onclick = saveOptions;
 });
 
+let updateLocationEl = document.getElementById('update-location');
+updateLocationEl.onclick = function () {
+	updateLocationEl.textContent = "Validating...";
+	updateLocationEl.disabled = true;
+
+	let zip = document.getElementById('zip-code').value;
+	let country = document.getElementById('country-code').value;
+	if (zip == '') {
+		responseMessage('You must specify a zip code.');
+		return;
+	}
+	if (country == '') {
+		responseMessage('You must specify a country code.');
+		return;
+	}
+
+	let appid = "e7f97bd1900b94491d3263f89cbe28d6";
```

Comment:
May not be a big deal or blocker, but we could probably hide this key. Or, have a "production" key.

Reply:
Yeah, I'll work on a solution to that next. This key's been bundled with this extension for quite some time.